### ISSUE TITLE
Firebase has a flat trajectory 

### ISSUE THREAD

**Issue by ExiLe-ZH** [TheSuperHackers/GeneralsGamePatch#960]
Just a visual issue, the cannon is aimed up further than the angle at which the shell leaves the barrel. This looks bad:

https://user-images.githubusercontent.com/32326277/186125819-d44c4d8d-08b7-48a5-ba58-6384760a5eaf.mp4

**Comment by xezon** [comment]
Can we get a video for it?

**Comment by ExiLe-ZH** [comment]
Added.

Think fixing the cannon's angle would be better than changing the trajectory, if possible.

**Comment by xezon** [comment]
I don't agree with changing the model. It will have more visual impact and is also less easy to tweak, as one would likely need to touch W3D model(s). Simply adjusting the shell trajectory is preferred if it is possible to retain its time from launch to hit.

**Comment by ExiLe-ZH** [comment]
Impact speed/reliability was my main concern, hence the suggestion. If it doesn't significantly change that, then fixing trajectory is fine too.

**Comment by Stubbjax** [comment]
Here's ShockWave's version for reference:

https://user-images.githubusercontent.com/11547761/186140891-293fe3e0-7eaf-42f4-a67b-6db1fc2e3b30.mp4

**Comment by xezon** [comment]
The tracer is nice. Makes it easier to see. It is quite large though. Looks like that model is aimed much lower.

**Comment by ExiLe-ZH** [comment]
Angle and trajectory match alot better, though I like the simple particle fx from zh more.

**Comment by ZekeDlyoung** [comment]
I remember in the past some people did complain about ShockWave's Firebase as it changed the Firebase Shell functionality from a missile (yes you heard that right) which gave it fast speed and accuracy, to a dumb projectile (same as inferno) which made it look and feel better, but made it miss more often inadvertently nerfing it.

I am neutral on this, as I understand most people would like to keep its performance over looks. I'm just posting this as more information for the decision making process.

Also, no, you can't really keep it as a missile with the same functionality and still make it arc beautifully. One side will have to be compromised for the other, or at the very least a lot of tweaking needs to be done to get both

**Comment by ZekeDlyoung** [comment]
Nvm, I was thinking of the Strategy Center shell 🙃
